README.md

```diff
@@ -218,3 +219,76 @@ than the current reputation.
 * Successful response status code: 204 No Content
 
 Example: `curl -d '{"Violation": "password-check-rate-limited-exceeded"}' -X PUT http://tigerblood/violations/240.0.0.1 --header "Authorization: {YOUR_HAWK_HEADER}"`
+
+#### PUT /violations/
+
+Sets or updates the reputations for multiple IP addresses or networks
+with provided violation types i.e. `PUT /violations/{ip}` for each IP.
+
+Accepts duplicates of an IP.
+
+In the event of an invalid or failed entry, returns the failing entry
+and index with the error response body below and does not roll back
+the accepted entries (i.e. doesn't run in a transaction).
```

Comment:
I wanted to avoid having to return a bunch of error messages for requests with multiple bad entries, but going with the single prepared statement route might simplify that.